packages/@ama-sdk/core/src/fwk/errors.ts

```diff
@@ -16,10 +18,10 @@ interface GenericErrorContext {
 export class GenericError extends Error {
 
   constructor(message: string, context?: GenericErrorContext) {
```

Comment:
Maybe you should turn the `context` field as `public` to have it in the error serialization

Reply:
I have done some changes from the previous implementation, and it does not seem to be needed to be serialized.